frontend/src/metabase/components/DatabaseDetailsForm.jsx

```diff
@@ -266,7 +266,7 @@ export default class DatabaseDetailsForm extends Component {
             <div className="px2">
               <h3>{t`Use DNS SRV when connecting`}</h3>
               <div style={{ maxWidth: "40rem" }} className="pt1">
-                {t`Using this option requires that provided host is a FQDN.  If connecting to 
+                {t`Using this option requires that provided host is a FQDN.  If connecting to
```

Comment:
FYI I think in general it's best to not have extra whitespace in tagged strings since it appears in the `.pot` + POEditor, and changing whitespace invalidates previous translations (I think?)
![Screenshot 2019-09-06 10 51 51](https://user-images.githubusercontent.com/18193/64449272-5d443880-d094-11e9-84fe-deb194cf9788.png)

Reply:
Got it, I think this is in the diff due to `prettier` getting rid of that trailing space as opposed to something I was intentionally doing but I'll clean up those breaks while I'm in here to stop it looking like that.